influxdb_client/client/_base.py

```diff
@@ -111,11 +113,11 @@ def _from_config_file(cls, config_file: str = "config.ini", debug=None, enable_g
                 is_json = True
 
```

Comment:
Please use `kwargs` instead of new argument:

```suggestion
config_name = kwargs.get('config_name', 'influx2')
```